### PR TITLE
Ingestion pipeline by service type task

### PR DESCRIPTION
**Pull request by 07Himank**
I worked on ... because ... added filter for fetching ingestion pipeline by pipelineType.

### CODE REVIEW

File: openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/ListFilter.java [hunk 1]
```diff
@@ -126,6 +133,18 @@ private String getWebhookTypePrefixCondition(String tableName, String typePrefix
         : String.format("%s.webhookType LIKE '%s%%'", tableName, typePrefix);
   }
 
+  private String getPipelineTypePrefixCondition(String tableName, String pipelineType) {
+    pipelineType = escape(pipelineType);
+    if (DatasourceConfig.getInstance().getDriverClass().startsWith("com.mysql.cj.jdbc")) {
```
Comment (nahuelverdugo) [inline openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/ListFilter.java:138]: We are hiding some logic here, can we move this into a method of `DatasourceConfig` called `isMySQLConnector()` or something like this?

**Review by nahuelverdugo — COMMENTED** [review]
The PR description is empty, it is hard to know what are we reviewing or if it is part of some issue.

**Comment by 07Himank** [comment]
> The PR description is empty, it is hard to know what are we reviewing or if it is part of some issue.



> The PR description is empty, it is hard to know what are we reviewing or if it is part of some issue.

In this Pr we had added the filter to fetch the ingestion pipeline by pipelineType for eg:-pipelineType=dataInsight

File: openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/ListFilter.java (hunk 1 above)
Comment (07Himank) [reply openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/ListFilter.java]: Thank you for your suggestion. I have updated the code.

**Review by nahuelverdugo — APPROVED** [review]

**Pull request merged**